### PR TITLE
feat: user tools operator k8s scheduling

### PR DESCRIPTION
**Pull request by alpiquero**
Adds pod scheduling capabilities to an object of _usertools_ kind.

Must be merged after https://github.com/konstellation-io/kdl-server/pull/700

### CODE REVIEW

**Comment by alpiquero** [comment]
PR #700 merged. This PR can be merged now.

File: helm/kdl-server/crds/user-tools-operator-crd.yaml [hunk 1]
```diff
@@ -21,6 +21,876 @@ spec:
           spec:
```
Comment (aacecandev) [inline helm/kdl-server/crds/user-tools-operator-crd.yaml:21]: This comment is not related to this line! Can we please change the `metada.name` and `spec.group` properties to not anymore referencing science-toolkit?

**Review by aacecandev — APPROVED** [review]

File: helm/kdl-server/crds/user-tools-operator-crd.yaml (hunk 1 above)
Comment (alpiquero) [reply helm/kdl-server/crds/user-tools-operator-crd.yaml:21]: Done

**Review by blopezpi — APPROVED** [review]
